cli/tauri-bundler/src/main.rs

```diff
@@ -1,17 +1,26 @@
 mod bundle;
 mod error;
+mod sign;
+
 pub use error::{Error, Result};
 
-use crate::bundle::{bundle_project, check_icons, PackageType, Settings};
+use crate::bundle::{bundle_project, check_icons, BuildArtifact, PackageType, Settings};
```

Comment:
`BuildArtifact` seems to come from nowhere, I had to remove it to install Tauri bundler.